packages/xo-web/src/common/intl/messages.js

```diff
@@ -2393,6 +2393,7 @@ const messages = {
   trialLicenseInfo: 'You are currently in a {edition} trial period that will end on {date, date, medium}',
 
   // ----- proxies -----
+  deployProxyDisabled: 'Disabled for source users',
```

Comment:
@olivierlambert This is the tooltip that sources users will see on the disabled "Deploy a proxy" button, is that ok?